packages/Toast/index.js

```diff
@@ -0,0 +1,42 @@
+import React, { cloneElement, useContext } from 'react'
+import { ThemeContext, ThemeProvider } from '@xstyled/styled-components'
+import toast from 'toasted-notes'
+
+import * as S from './styles'
+
+export function useToast() {
+  const themeContext = useContext(ThemeContext)
+
+  function createToast(children, options) {
+    const getOptions = {
```

Comment:
```suggestion
    const toastOptions = {
```

(Ce n'est pas un fonctionne donc on a pas besoin de `getXxx`)